src/Symfony/Bridge/Monolog/Handler/FingersCrossed/HttpCodeActivationStrategy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Handler\FingersCrossed;
+
+use Monolog\Handler\FingersCrossed\ErrorLevelActivationStrategy;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Activation strategy that ignores certain HTTP codes.
+ *
+ * @author Shaun Simmons <shaun@envysphere.com>
+ */
+class HttpCodeActivationStrategy extends ErrorLevelActivationStrategy
+{
+    private $excludedCodes;
+    private $requestStack;
+
+    public function __construct(RequestStack $requestStack, array $excludedCodes, $actionLevel)
+    {
+        parent::__construct($actionLevel);
+
+        $this->requestStack = $requestStack;
+        $this->excludedCodes = $excludedCodes;
+    }
+
+    public function isHandlerActivated(array $record)
+    {
+        $isActivated = parent::isHandlerActivated($record);
+
+        if (
+            $isActivated
+            && isset($record['context']['exception'])
+            && $record['context']['exception'] instanceof HttpException
+            && ($request = $this->requestStack->getMasterRequest())
```

Comment:
`$request` is not used

Reply:
I think you are wrong, `$request` is used here: https://github.com/symfony/symfony/pull/23707/files#diff-ca46116b05ee5c3e47d1438e580c53eeR53

Reply:
It wasn't being used before. It is now with the addition of URL blacklisting.